dropwizard-jersey/src/main/java/io/dropwizard/jersey/DropwizardResourceConfig.java

```diff
@@ -122,16 +123,30 @@ public String getEndpointsInfo() {
         msg.append("The following paths were found for the configured resources:");
         msg.append(NEWLINE).append(NEWLINE);
 
-        final Set<Class<?>> allResources = new HashSet<>();
+        final Set<Class<?>> allResourcesClasses = new HashSet<>();
         for (Class<?> clazz : allClasses()) {
             if (!clazz.isInterface() && Resource.from(clazz) != null) {
-                allResources.add(clazz);
+                allResourcesClasses.add(clazz);
             }
         }
 
-        for (Class<?> klass : allResources) {
+        for (Class<?> klass : allResourcesClasses) {
             new EndpointLogger(urlPattern, klass).populate(endpointLogLines);
         }
+        
+        final Set<Resource> allResources = this.getResources();
+        for (Resource res : allResources) {
+            if (containsAny(allResourcesClasses, res.getHandlerClasses())) {
```

Comment:
I think we could avoid creating a helper method and use a declarative approach with Guava's  `Iterables`.
```
Iterables.any(res.getHandlerClasses(), allResourcesClasses::contains)
```